Dev

Letrobit opened the pull request:
PULL REQUEST

Letrobit committed:
First attempt - first commit

egorvas commented on pom.xml:
```diff
@@ -5,14 +5,32 @@
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```
**нужно исправить:** нужно использовать одну версию junit 4 или 5, а у тебя в pom указаны обе

egorvas commented on src/test/java/TestLionWithParameters.java:
```diff
@@ -0,0 +1,43 @@
+    private String sex;
+    private boolean result;
+
+    Feline feline = Mockito.mock(Feline.class);
```
**Нужно исправить:** я не понимаю зачем тут мок и какую функцию он выполняет. По условию заданию тебе нужно как-то использовать моки, а они у тебя просто есть.

Letrobit committed:
Second commit
Fixed: deleted dependancy Junit 5 (replaced w/ Junit 4 in tests)
Added mock usability in all necessary tests
Jacoco: 100% coverage, unchanged
